rust/lance/src/dataset/write/commit.rs

```diff
@@ -311,6 +328,15 @@ impl<'a> CommitBuilder<'a> {
                 location: location!(),
             });
         } else {
+            info!(target: TRACE_DATASET_EVENTS, event=CREATE_EVENT, path=&base_path.to_string());
```

Comment:
Here we can catch `create` event. A little tricky is: if users create a dataset and then write in one step, there would be two data events, and the creation would be behind the writing.

Reply:
Use `ing` and `ed` to distinguish them.